README.md

```diff
@@ -96,7 +96,7 @@ For an overview of the project's architecture please check out our [ARCHITECTURE
 
 ---------------
 
-For a deep dive in Unity Netcode and Boss Room, visit our [doc](https://docs-multiplayer.unity3d.com/) and [Unity Multiplayer Resources](https://docs-multiplayer.unity3d.com/netcode/current/learn/introduction) sections.
+For a deep dive in Unity Netcode and Boss Room, visit our [docs site](https://docs-multiplayer.unity3d.com/).
```

Comment:
```suggestion
For a deep dive in Unity Netcode and Boss Room, visit our [docs site](https://docs-multiplayer.unity3d.com/) and [Boss Room section](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom). Please make sure to visit our [Concepts and FAQ section](https://docs-multiplayer.unity3d.com/netcode/current/reference/glossary/high-level-terminology)
```

Reply:
I kept it on the bare side due to more docs changes potentially breaking structure, therefore links.